src/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { LinearGradient } from 'expo-linear-gradient';
+import {
+    StyleSheet,
+    Text,
+    View,
+    TouchableOpacity,
+    ImageBackground,
+  } from 'react-native';
+
+
+  export const LandingPage = () => {
+    return (
+      <View style={styles.container}>
+         <ImageBackground
+          source={require('../../assets/Background.png')}
+          style={styles.backgroundImg}
+        >
+          <LinearGradient
+            colors={['rgba(196, 196, 196, 0.66)', 'rgba(196, 196, 196, 0)']}
+            style={styles.linearGradient}
+          >
+            <Text style={styles.logoHeader}>
+              {'Hifu'.toUpperCase()}
+            </Text>
+            <Text style={styles.logoText}>
+              {'When you want to get lost but still found'.toUpperCase()}
+            </Text>
+            <TouchableOpacity 
```

Comment:
Touchable opacity — interested to see what this is all about.

Reply:
Yeah! It's a super cool part of react native that you can use/ import for the buttons that you want to be able to be clickable/ interactive for you site. Here are the docs for it: 
[Touchable](https://reactnative.dev/docs/touchableopacity)